Description: Do not strip the debugging symbols from the *.oct files
 Also, include the -g option, to enable debugging option for all
 compilers.
Author: Rafael Laboissiere <rafael@debian.org>
Forwarded: no
Last-Update: 2018-12-30

--- octave-octproj-1.1.5.orig/src/Makefile
+++ octave-octproj-1.1.5/src/Makefile
@@ -21,11 +21,11 @@ all: compile
 compile:
 	$(MKOCTFILE) -c projwrap.c -o projwrap.o
 	$(MKOCTFILE) -c geodgeoc.c -o geodgeoc.o
-	$(MKOCTFILE) -s _op_fwd.cc projwrap.o $(LDFLAGS)
-	$(MKOCTFILE) -s _op_inv.cc projwrap.o $(LDFLAGS)
-	$(MKOCTFILE) -s _op_transform.cc projwrap.o $(LDFLAGS)
-	$(MKOCTFILE) -s _op_geod2geoc.cc geodgeoc.o
-	$(MKOCTFILE) -s _op_geoc2geod.cc geodgeoc.o
+	$(MKOCTFILE) -g _op_fwd.cc projwrap.o $(LDFLAGS)
+	$(MKOCTFILE) -g _op_inv.cc projwrap.o $(LDFLAGS)
+	$(MKOCTFILE) -g _op_transform.cc projwrap.o $(LDFLAGS)
+	$(MKOCTFILE) -g _op_geod2geoc.cc geodgeoc.o
+	$(MKOCTFILE) -g _op_geoc2geod.cc geodgeoc.o
 
 .PHONY: clean
 clean:

